src/utils/mapper/code-generator.ts

```diff
@@ -19,6 +19,125 @@
 import generate from "@babel/generator";
 import { File } from "@babel/types";
 
+/**
+ * Generate code from and AST(Abstract Syntax Tree).
+ *
+ * {@link https://babeljs.io/docs/en/babel-generator}
+ *
+ * @example
+ * // returns "var a = 2"
+ * generateCodeFromAst({
```

Comment:
Shall we shorten the example.

```suggestion
 * generateCodeFromAst(<CODE>);
```

Reply:
Fixed with [2c1ee22]()